docs/maui/views/popup-service.md

```diff
@@ -182,11 +207,11 @@ Expanding on the previous example the following implementation can be added to t
 [RelayCommand(CanExecute = nameof(CanSave))]
 void OnSave()
 {
-    popupService.ClosePopup(Name);
+    popupService.ClosePopupAsync(Shell.Current.Navigation, Name);
 }
 ```
```

Comment:
This needs to be an async method and we need to await ClosePopupAsync. For simplicity, let's use async/await instead of returning a Task, which is also valid here.

We can use `Shell.Current` instead of `Shell.Current.Navigation`, however, I think it's totally cool to keep `Shell.Current.Navigation` since it highlights that devs can pass in `INavigation` as a valid parameter.

```cs
[RelayCommand(CanExecute = nameof(CanSave))]
 async Task OnSave()
 {
     await popupService.ClosePopupAsync(Shell.Current.Navigation, Name);
 }
 ```